### PR TITLE
深色模式调整题目图片颜色

### PR DESCRIPTION
**Pull request by zhouyiqing0304**
## Thank you for your contribution to the XMOJ-Script Community!

Please read the comments below to help us consider your Pull Request.

We are all volunteers and completing the process outlined will help us review your commits quicker.

Please use this pull request template, or your pull request will be closed.

**Please make sure you**

1. Base your code and PRs against the repositories developmental branch.

### When you open a pull request, please delete the text above.
---

**What does this PR aim to accomplish?:**

**How does this PR accomplish the above?:**

---
**By submitting this pull request, I confirm the following:**

1. I have read and understood the [contributors guide](https://github.com/XMOJ-Script-dev/XMOJ-Script/blob/dev/CONTRIBUTING.md), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [GNU General Public License v3.0](https://github.com/XMOJ-Script-dev/XMOJ-Script/blob/dev/LICENSE)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
6. I have checked that another pull request for this purpose does not exist.
7. I have considered, and confirmed that this submission will be valuable to others.
8. I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
9. I give this submission freely, and claim no ownership to its content.

---
- [x] I have read the above and my PR is ready for review. *Check this box to confirm*
- [x] 我承认这个功能很抽象

### CODE REVIEW

**Comment by zhouyiqing0304** [comment]
@boomzero @PythonSmall-Q 
你们看看，不喜勿喷

**Comment by zhouyiqing0304** [comment]
image

**Comment by zhouyiqing0304** [comment]
image
Edge浏览器有这个功能

**Comment by PythonSmall-Q** [comment]
@boomzero 你评判吧，我不用深色模式的

**Comment by zhouyiqing0304** [comment]
@PythonSmall-Q 你都不觉得这个功能抽象吗

**Comment by PythonSmall-Q** [comment]
没仔细看，也不想仔细看。看着有点无语

**Comment by zhouyiqing0304** [comment]
我自己都无语

**Comment by zhouyiqing0304** [comment]
微软抽象，我脑子也指定有问题

**Comment by boomzero** [comment]
> ### When you open a pull request, please delete the text above.
...

File: XMOJ.user.js [hunk 1]
```diff
@@ -1183,7 +1208,8 @@ async function main() {
                         }, {"ID": "ResetType", "Type": "F", "Name": "重新排版*"}, {
                             "ID": "AddColorText", "Type": "A", "Name": "增加彩色文字"
                         }, {"ID": "AddUnits", "Type": "A", "Name": "状态界面内存与耗时添加单位"}, {
-                            "ID": "DarkMode", "Type": "A", "Name": "使用暗色模式"
+                            "ID": "DarkMode", "Type": "A", "Name": "使用暗色模式"}, {
+                            "ID": "DarkPicture", "Type": "A", "Name": "使用反色的题目图片（开发中，可能不稳定；需要启用暗色模式）"
```
Comment (boomzero) [inline XMOJ.user.js:1212]: remove `开发中，可能不稳定`
If it is unstable, disable it by default

**Review by boomzero — CHANGES_REQUESTED** [review]

File: XMOJ.user.js (hunk 1 above)
Comment (boomzero) [reply XMOJ.user.js:1212]: But otherwise okay

Comment (PythonSmall-Q) [reply XMOJ.user.js]: done，他一直不在线

**Review by PythonSmall-Q — APPROVED** [review]

**Review by boomzero — APPROVED** [review]

**Comment by boomzero** [comment]
Okay

**Comment by boomzero** [comment]
lgtm

**Comment by boomzero** [comment]
@PythonSmall-Q sync to dev

**Comment by boomzero** [comment]
> @PythonSmall-Q sync to dev

先不要merge进去， 我要改一个东西

**Comment by PythonSmall-Q** [comment]
改好了吗

**Comment by boomzero** [comment]
Reverted due to #627

**Pull request merged**